create firs example (#1)
mpodanev/lm-screenshoter: README.md

# How to use

> First of all, should install packages `npm ci` and `backstop init`.

Reviewer: Please read how to run local version of package without global instalation.

Reviewer 2: Добавил. Из node_modules можно запускать

For the run application you should use the next example command:

`nmp start -- --test --project="project-name"`

where:

> * `--test` -- creates a set of test screenshots and compares them with your reference screenshots. You can use short command `-t`.

Reviewer: ```suggestion
* `--test` -- creates a set of test screenshots and compares them with your reference screenshots. You can use an alias `-t`.
```

* `--approve` -- if the test you ran looks good, then go ahead and approve it. Approving changes will update your
 reference files with the results from your last test. Future tests are compared against your most recent approved

>   test screenshots. You can use a short command `-a`.

Reviewer: ```suggestion
  test screenshots. You can use an alias `-a`.
```

> * `--project` -- set `project-name` to specify the project folder you'd like to test. You can use a short command `-p

Reviewer: ```suggestion
* `--project` -- set `project-name` to specify the project folder you'd like to test. You can use an alias `-p
```

`. Also, you can use parameter without equals symbol like this: `-p project-name`.

In `index.js` file you need set absolute path in `projectsDirectoryPath` parameter for projects path.

> ###Examples

Reviewer: ```suggestion
##Examples
```
